app/js/actions/jobs.js

```diff
@@ -55,14 +54,14 @@ export const linkInputArtifact = (input, artifacts) => ({
     artifacts
 });
 
-export const setJob = (action, actionType) => {
+export const setJob = action => {
```

Comment:
I actually like the parens, so I added them back (it's less work than changing all of them(